src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -255,6 +255,8 @@
                     BaseRegistry="$(ContainerBaseRegistry)"
                     BaseImageName="$(ContainerBaseName)"
                     BaseImageTag="$(ContainerBaseTag)"
+                    BaseImageDigest="$(ContainerBaseDigest)"
```

Comment:
The errors in the pipeline says smth about BaseImageDigest. I think removing it will fix it

Reply:
Thanks - I've done so now.